### DISCUSSION TITLE
io/shangtou

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#184] [Announcements]
# io/shangtou

Java程序员进阶之路，小白的零基础Java教程，Java IO 体系看起来类很多，感觉很复杂，但其实是 IO 涉及的因素太多了。在设计 IO 相关的类时，编写者也不是从同一个方面考虑的，所以会给人一种很乱的感觉，并且还有设计模式的使用，更加难以使用这些 IO 类，所以特地对 Java 的 IO 做一个总结。

https://tobebetterjavaer.com/io/shangtou.html

**Comment by dark-eternity** [comment]
还行
